packages/chamo/chamo.4.0/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+synopsis: "A kind of emacs-like editor, using OCaml instead of lisp"
+maintainer: "zoggy@bat8.org"
+authors: "Zoggy <zoggy@bat8.org>"
+license: "GPL-3.0-only"
+homepage: "https://zoggy.frama.io/chamo/"
+doc: "https://zoggy.frama.io/chamo/doc.html"
+bug-reports: "https://framagit.org/zoggy/chamo/issues"
+depends: [
+  "dune" {>= "3.6"}
+  "ocaml" {>= "4.12.0"}
+  "ocaml-compiler-libs"
```

Comment:
```suggestion
```
This package is different from `compiler-libs`. This is a janestreet wrapper around compiler-libs that you're not using.

Reply:
ah, thanks.